src/main/java/net/sf/rails/util/GameLoader.java

```diff
@@ -42,6 +46,20 @@ public static void loadAndStartGame(File gameFile) {
         SplashWindow splashWindow = new SplashWindow(true, gameFile.getAbsolutePath());
         splashWindow.notifyOfStep(SplashWindow.STEP_LOAD_GAME);
 
+        // check to see if we were passed in a last_rails file
+        if ( GameUIManager.DEFAULT_SAVE_POLLING_EXTENSION.equals(StringUtils.substringAfterLast(gameFile.getName(), ".")) ) {
+            // read the filename from the last rails file
+            log.debug("loading current game file from last_rails {}", gameFile);
+            try {
+                String gameFileStr = FileUtils.readFileToString(gameFile, StandardCharsets.ISO_8859_1).trim();
+                gameFile = new File(gameFile.getParentFile(), gameFileStr);
+            }
+            catch (IOException e) {
+                log.warn("unable to load {}", gameFile);
+                return;
```

Comment:
What does the application do in this case?

Reply:
Not my code-- haven't reviewed